modules/programs/dircolors.nix

```diff
@@ -43,6 +43,42 @@ in {
       '';
     };
 
+    terminalTypes = mkOption {
+      type = with types; listOf str;
+      # Default terminal types from `dircolors --print-database`.
+      default = [
+        "Eterm"
+        "ansi"
+        "*color*"
+        "con[0-9]*x[0-9]*"
+        "cons25"
+        "console"
+        "cygwin"
+        "dtterm"
+        "gnome"
+        "hurd"
+        "jfbterm"
+        "konsole"
+        "kterm"
+        "linux"
+        "linux-c"
+        "mlterm"
+        "putty"
+        "rxvt*"
+        "screen*"
+        "st"
+        "terminator"
+        "tmux*"
+        "vt100"
+        "xterm*"
+      ];
```

Comment:
Shouldn't the default be empty, instead of having to synchronize with `dircolors`?